README.md

```diff
@@ -42,6 +42,21 @@ As CGRA-Flow requires GUI, a script is provided for setting up the display:
  # Don't forget to activate the python virtual environment once you are in the container:
  source /WORK_REPO/venv/bin/activate
 ```
+
+```shell
+# Windows Docker customtkinter style UI (Please setup GUI (X-11) first)
+# In WSL
+docker pull yuqisun/cgra-flow-win:latest
+docker run -it yuqisun/cgra-flow-win bash
+docker run -it --name cgra-flow-win yuqisun/cgra-flow-win bash
+
+# In container
+root@2cd33efd97b3 WORK_REPO# export DISPLAY=192.168.3.38:0.0 # change to your own ipv4 address with 0.0
+root@2cd33efd97b3 WORK_REPO# source venv/bin/activate
+(venv) root@2cd33efd97b3 WORK_REPO# cd CGRA-Flow/
+(venv) root@2cd33efd97b3 CGRA-Flow# python launchUI_custom.py
```

Comment:
Should be mkdir build, then cd, then launch?

Reply:
ah, let me update this

Reply:
updated